src/etc/inc/filter.inc

```diff
@@ -3397,6 +3397,7 @@ EOD;
 				}
 				/* allow access to DHCP relay on interfaces */
 				if (isset($config['dhcrelay']['enable'])) {
+					$dhcrelaycfg =& $config['dhcrelay'];
 					$dhcifaces = explode(",", $dhcrelaycfg['interface']);
```

Comment:
Well, the only reason is that it's being done everywhere else. And the variable is being completely unnecessary everywhere else as well.

```
$ grep -Rni dhcrelaycfg /etc/inc | grep "dhcrelaycfg ="
/etc/inc/services.inc:1726:     $dhcrelaycfg =& $config['dhcrelay'];
/etc/inc/services.inc:1860:     $dhcrelaycfg =& $config['dhcrelay6'];
/etc/inc/upgrade_config.inc:2556:       $dhcrelaycfg =& $config['dhcrelay'];
```

The scope of this PR is a oneliner fix for an issue I keep hitting for years. Frankly, I do not care much about how's it done but would love to have the thing finally working properly,